diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -315,24 +358,14 @@ function StudentQuiz(runtime, element, initData) {
       } else {
         if (currentIndex > newIndex) {
           // allow to move backwards without validate & save
-          return true;
-        }
-        var selectedChoice = $(visibleAnswerChoice, element).find(selectedStudentChoice).val();
-
-        if (selectedChoice !== "" && selectedChoice !== undefined) {
-          return submitQuestionResponse(isLast, currentStep);
-        } else {
-          //log event for question validation error
           common.publishEvent({
-            event_type: 'xblock.diagnostic_feedback.quiz.question.error',
-            quiz_type: initData.quiz_type,
-            quiz_title: initData.quiz_title,
-            current_question: currentStep,
-            message: gettext('Please select an answer')
+            event_type: 'xblock.diagnostic_feedback.quiz.question.reloading',
+            reloading_question: newIndex
           });
-          common.showStudentValidationError({success: false, warning: false, msg: gettext('Please select an answer')});
```

Comment:
@jareerahsan Are you sure you want to remove this part completely? I'm not really familiar with how the validation mechanisms for this XBlock work, but after these changes `showStudentValidationError` is not used at all anymore. If you do want to deprecate it completely, please remove its definition from public/js/src/common.js.